examples/promises/deferred.php

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env php
+<?php
+
+use Amp\Deferred;
+use Amp\Loop;
+
+require_once __DIR__ . "/../../vendor/autoload.php";
+
+/**
+ * @return \Amp\Promise<string>
+ */
+function jobSuccess() {
+    $deferred = new Deferred();
+
+    //we delay Promise resolve for 1 sec to simulate some async job
```

Comment:
Please add a space after `//` and capitalize the `We` and add a period after `job`.